src/send-query.js

```diff
@@ -0,0 +1,12 @@
+import ajax from './ajax';
+
+export default function sendQuery(query, config) {
+  const url = `https://${config.domain}/api/graph`;
+  const headers = {
+    authorization: `Basic ${btoa(config.apiKey)}`,
+    'content-type': 'application/json'
+  };
+  const body = JSON.stringify({ query });
```

Comment:
Should there be some checking on the query or the config? I am not sure how public send-query so I am not sure how much it will matter.